### DISCUSSION TITLE
Flagged account; 404 to other people trying to open it.

### DISCUSSION THREAD

**Discussion by tsvt1n** [community/community#86734] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I need someone to assist with giving information on the status of ticket #2509248 ; One of the github bots did a reset on my password for security reasons (i'm not sure why). After I generated a new password, found out my account is flagged as 'SPAMMY' and when other people try to open it they get 404. The ticket is open for more than 7 days and it says '...it may take up to seven business days to hear back from our Support Engineers.', so I'm a bit worried if it got lost somewhere down the line. Will appreciate any feedback - Thank you!
my original profile is @tsvtln : https://github.com/tsvtln

**Comment by queenofcorgis** [comment, marked as answer]
Hi @tsvt1n, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
